src/Data/Validation.hs

```diff
@@ -349,24 +333,14 @@ instance Validate Either where
 _Failure ::
   Validate f =>
   Prism (f e1 a) (f e2 a) e1 e2
-_Failure =
-  prism
-    (\x -> _Either # Left x)
-    (\x -> case x ^. _Either of
-             Left e -> Right e
-             Right a -> Left (_Either # Right a))
-{-# INLINE _Failure #-}
+_Failure = _Either . _Left
+{-# INLINE [1] _Failure #-}
```

Comment:
Oops, this inline staging was an artifact of experiments. That line should go back to how it was.